pyroma/pypidata.py

```diff
@@ -1,50 +1,51 @@
 import tempfile
 import os
 import logging
-from pyroma import distributiondata
+import requests
+
+from xmlrpc import client
+from urllib import request
```

Comment:
```suggestion
```

This import appears to be no longer used, and may confuse users and autocomplete with `requests` and lead to hard to spot typo bugs, so I'd suggest removing it.